spk/bazarr/Makefile

```diff
@@ -0,0 +1,42 @@
+SPK_NAME = bazarr
+
+SPK_VERS = 0.9.1.1
+SPK_REV = 1
+
+SPK_ICON = src/bazarr.png
+
+BUILD_DEPENDS = cross/python38
+
+DEPENDS = cross/$(SPK_NAME)
+
+WHEELS = 
+SPK_DEPENDS = "python38:ffmpeg"
```

Comment:
Sure about it? isn't it "python38,ffmpeg" ?

Reply:
all packages have `:` separator for SPK_DEPENDS, and so have synology.com packages

Reply:
To be honest it seemed to work on my side and it seemed to be the same for other spks so I didn't go any further than this.